Tests/Test_Sim.c

```diff
@@ -0,0 +1,46 @@
+//#include "Simulator.h"
+#include "Amps.h"
+#include "common.h"
+#include "config.h"
+#include "CANbus.h"
+#include "os.h"
+#include "Tasks.h"
+#ifndef SIMULATION
+#include "stm32f4xx.h"
+#else
+#include "Simulator.h"
+#endif
+#include "BSP_Lights.h"
+#include "BSP_PLL.h"
+#include "CAN_Queue.h"
+#include "BSP_WDTimer.h"
+#include "BSP_Contactor.h"
+
+/**
```

Comment:
It seems like this test file is the main one for testing current simulation, but it looks like you made 3 test files total for current simulation. I'm a bit confused about what specifically is being tested in each one/ the purpose of each one.